test/verifiedTests/testGDLS/testGDLS.m

```diff
@@ -0,0 +1,36 @@
+function statusOK = testGDLS()

```

Comment:
This file should be removed from this PR. This is unrelated to the progress bar, Open a separate PR.